index.html

```diff
@@ -0,0 +1,168 @@
+  <body>
+    <header class="header">
+      <section class="nav-content"
+        ><h6>&nbsp;</h6>
+        <nav class="navigation">
+          <a class="item item__left-most" href="#">
+            <img
+              src="./img/apple_logo_small.png"
+              alt="Apple logo"
+              class="apple_logo"
+            />
+          </a>
+          <a class="item" href="#">
+            Store
+          </a>
+          <a class="item" href="#">Mac</a>
+          <a class="item" href="#">iPod</a>
+          <a class="item" href="#">iPhone</a>
+          <a class="item" href="#">iPad</a>
+          <a class="item" href="#">iTunes</a>
+          <a class="item" href="#">Support</a>
+          <div class="item item__right-most">
+            <form class="search-form">
+              <div class="search-form__input">
+                <i class="fa fa-search"></i>
+              </div>
+              <input type="text" class="search-form__input-search" />
+            </form>
+          </div>
+        </nav>
+      </section>
+      <section class="header__section">
+        <div class="header__content">
+          <h1 class="display-2 header__title">What will your verse be?</h1>
+          <h3 class="display-5 header__desc"
+            >Everyone has something to share. See how people are using iPad in
+            their unique ways to add to the world’s story.</h3
+          >
+          <h5 class="display-5 header__video-link"
+            >Watch the “Your Verse” film and explore the stories</h5
+          >
+        </div>
+      </section>
+    </header>
+    <section class="featured"
+      ><h6>&nbsp;</h6>
+      <div class="featured__item">
+        <a href="#" class="featured__item__1-link">
+          <img
+            src="./img/promo_iphone5s_2x.jpg"
+            alt=""
+            class="featured__item__ featured__item__1"
+          />
+        </a>
+      </div>
+      <div class="featured__item">
+        <a href="#" class="featured__item__2-link">
+          <img
+            src="./img/promo_iphone5c_2x.jpg"
+            alt=""
+            class="featured__item__ featured__item__2"
+          />
+        </a>
+      </div>
+      <div class="featured__item">
+        <a href="#" class="featured__item__3-link">
+          <img
+            src="./img/promo_30_years_2x.jpg"
+            alt=""
+            class="featured__item__ featured__item__3"
+          />
+        </a>
+      </div>
+      <div class="featured__item">
+        <a href="#" class="featured__item__4-link">
+          <img
+            src="./img/promo_supplier_responsibility_2x.jpg"
+            alt=""
+            class="featured__item__ featured__item__4"
+          />
+        </a>
+      </div>
+    </section>
+    <footer class="footer">
+      <div class="footer__div footer__text">
+        <p class="footer__text-1">
+          Shop the Apple Online Store(1-800-MY-APPLE), visit an Apple Retail
+          Store, or find a reseller.
+        </p>
```

Comment:
'Apple Online Store', 'Apple Retail Store', 'reseller' - should be links

Reply:
done